Documentation/tpv13.rst

```diff
@@ -3,7 +3,9 @@
 SCEC TPV13
 ==========
 
-TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. TPV13 is using non-associative Drucker-Prager plasticity with yielding in shear. The material is characterized by six constitutive parameters:
+TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. TPV13 is using non-associative Drucker-Prager plasticity with yielding in shear. To run TPV13, it requires to recompile SeisSol with "plasticity = 'yes'" in the configuration python script. 
```

Comment:
"non-associative Drucker-Prager plastic with yielding in shear".
Remove the 2nd sentence.